### PR TITLE
MAUI-2299 Provided UG documentation for SfBusyIndicator control in .NET MAUI

### PR DESCRIPTION
**Pull request by anandhan-rajagopal**
**Feature description**

Added UG for SfBusyIndicator control in .NET MAUI.

The following categories in UG documentation for BusyIndicator control

- Overview
- Getting Started
- Migration from Xamarin.Forms into .NET MAUI
- Make Busy Animation Idle
- Animation Type
- Set Header
- Duration
- Sizing
- Indicator color and Background

**Analysis and design**

Analyzed UG for BusyIndicator in Xamarin.Forms -  https://help.syncfusion.com/xamarin/busy-indicator/getting-started

Analyzed the API review task: https://syncfusion.atlassian.net/browse/MAUI-2141

**Test cases**

I have ensured the XAML and C# code outputs.

### CODE REVIEW

**Comment by anandhan-rajagopal** [comment]
rebuild

**Review by PaulAnderson02 — APPROVED** [review]
Fine

File: MAUI/Busy-Indicator/Migration.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<th>.NET MAUI SfBusyIndicator</th>
+<th>Description</th></tr>
+<tr>
+<td> IsBusy </td>
+<td>IsRunning</td>
+<td>Gets or sets a value indicating whether gets or sets the value of the IsBusy property in busy indicator. It determines whether the animation of indicator to be shown or not.</td></tr>
+<tr>
+<td>Duration</td>
+<td>DurationFactor</td>
+<td>Gets or sets the value of the Duration property. It used to change the speed of animation to finish a cycle.</td></tr>
```
Comment (florenceliliana) [inline MAUI/Busy-Indicator/Migration.md:39]: <td>Gets or sets the value of the Duration property. It is used to change the speed of animation to finish a cycle.</td></tr>

File: MAUI/Busy-Indicator/Migration.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+<tr>
+<td>ViewBoxWidth ViewBoxHeight</td>
+<td>SizeFactor</td>
+<td>Gets or set the size factor of the indicator. This is not applicable for all animation types.</td></tr>
+<tr>
+<td>OverlayColor</td>
+<td>OverlayFill</td>
+<td>Gets or sets the overlay fill brush</td></tr>
```
Comment (florenceliliana) [inline MAUI/Busy-Indicator/Migration.md:47]: <td>Gets or sets the overlay fill brush</td></tr>.

File: MAUI/Busy-Indicator/Set-Header.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+description: Learn here all about Set Header support in Syncfusion .NET MAUI Busy Indicator (SfBusyIndicator) control and more.
+platform: maui
+control: BusyIndicator
+documentation: ug
+---
+# Set Header in .NET MAUI Busy Indicator (SfBusyIndicator)
+
+## Set Header in SfBusyIndicator
+
+The Syncfusion Busy Indicator provides a feature to display a title with font customizations
```
Comment (florenceliliana) [inline MAUI/Busy-Indicator/Set-Header.md:13]: The Syncfusion Busy Indicator provides a feature to display a title with font customizations.

File: MAUI/Busy-Indicator/Set-Header.md [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+
+The following screenshot illustrates the result of the above code.
+
+![TitleSpacing](Images/SetHeader/Spacing.png)
+
+
+
+## Font customization
+
+SfBusyIndicator provides options to customize the title text.  The title text can be customized using the `FontSize`, `FontAttributes` and `FontFamily` properties.
```
Comment (florenceliliana) [inline MAUI/Busy-Indicator/Set-Header.md:259]: SfBusyIndicator provides options to customize the title text. The title text can be customized using the `FontSize,` `FontAttributes,` and `FontFamily` properties.

File: MAUI/Busy-Indicator/Sizing.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+description: Learn here all about Sizing support in Syncfusion .NET MAUI Busy Indicator (SfBusyIndicator) control and more.
+platform: maui
+control: BusyIndicator
+documentation: ug
+---
+# Sizing in .NET MAUI Busy Indicator (SfBusyIndicator)
+
+## Sizing in SfBusyIndicator
+
+Drawing size can be customized in SfBusyIndicator.  `SizeFactor` property can be used to set height and width of the Indicator.  Its default value is 0.5 and  range between 0 to 1.
```
Comment (florenceliliana) [inline MAUI/Busy-Indicator/Sizing.md:13]: Drawing size can be customized in SfBusyIndicator. `SizeFactor` property can be used to set the height and width of the Indicator. Its default value is 0.5 and ranges from 0 to 1.

File: MAUI/Busy-Indicator/IndicatorColor.md [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+            }
+        }
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+The `OverlayFill` property type is `Brush`, so can able set Gradients as background.
```
Comment (florenceliliana) [inline MAUI/Busy-Indicator/IndicatorColor.md:124]: The `OverlayFill` property type is `Brush,` so you can set Gradients as background.

File: MAUI/Busy-Indicator/Getting-Started.md [hunk 7]
```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title: Getting Started with .NET MAUI Busy Indicator control | Syncfusion
```
Comment (florenceliliana) [inline MAUI/Busy-Indicator/Getting-Started.md:3]: title: Getting Started with .NET MAUI Busy Indicator control | Syncfusion
description: Learn here about getting started with Syncfusion .NET MAUI Busy Indicator (SfBusyIndicator) control, its elements, and more.

**Comment by anandhan-rajagopal** [comment]
All of the review changes mentioned in this PR have been addressed, and changes have been made, PR: https://github.com/syncfusion-content/maui-docs/pull/356

**Pull request merged**